packages/nimble-components/src/anchor-tabs/specs/README.md

```diff
@@ -0,0 +1,204 @@
+# Anchor Tabs
+
+## Overview
+
+Anchor tabs are a sequence of tabs that navigate to URLs when activated.
+
+### Background
+
+[Visual Design spec](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/b2aa2c0c-03b7-4571-8e0d-de88baf0814b/specs/) (same visual design as non-anchor tabs)
+
+[GitHub issue](https://github.com/ni/nimble/issues/479)
+
+### Non-goals
+
+- Mixing regular tabs and anchor tabs together
+- Vertical tabs orientation
+
+### Features
+
+- Functions like an anchor (i.e. navigates) upon activation
+- Supports keyboard navigation within group of tabs without automatically activating each focused tab
+- Disable individual tabs
+
+### Risks and Challenges
+
+None
+
+### Prior Art/Examples
+
+In SLE:
+
+![SLE](sle-systems.png)
+
+In GitHub:
+
+![GitHub](github.png)
+
+---
+
+## Design
+
+Anchor tabs have the same appearance as a standard tabs control, i.e. a sequence of operable regions with labels, each of which corresponds to some target content to display. In the case of anchor tabs, they do not switch between pre-configured tab panels, but rather navigate to some URL. The tabs can be configured to load the URL in a specific target, like an `iframe`.
+
+The design consists of an "Anchor tabs" container component which contains one or more "Anchor tab" components. This mirrors the existing design of the `nimble-tabs` and `nimble-tab` components, however there is no analog for the `nimble-tab-panel` component. Instead, the content loaded from the configured URL(s) loads into the containing page, an `iframe`, a new window, etc.
+
+We cannot extend FAST's tabs control because it has baked into it the idea that tabs activate tab panels. Instead we will create new components from scratch, but we can likely borrow logic from FAST e.g. for handling keyboard navigation.
+
+### API
+
+#### Anchor Tabs
+
+- *Component Name* - `nimble-anchor-tabs`
+- *Props/Attrs*
+    -   `activeid` - id of active tab. Defaults to the first tab.
+- *Methods* - None
+- *Events* - None
+- *CSS Classes and CSS Custom Properties that affect the component* - None
+- *Slots*
+    -   `start` - content placed to the left of tab list
+    -   `anchortabs` - slot for anchor tab elements
```

Comment:
The "normal" tab's template specifies the `tab` slot, so I think we should stick with `anchortab` so that you cannot slot normal tabs into a `nimble-anchor-tabs` element.